### ISSUE TITLE
Serenity ARM support

### ISSUE THREAD

**Issue by iamlevik** [serenity-bdd/serenity-core#2497]
Hello,

 Are there any plans to support ARM architecture for M1 ?  

Running tests through the regular command line "mvn clean verify" , we can see that it opens the Google Chrome using Intel (x86_64) rather than the Apple version (ARM).  I have to mention that the correct chromedrivers and Google Chrome versions have been downloaded on the machine. 

During our tests using a standalone Python + Selenium, we opened both Chromedriver and the Google Chrome application using (Apple).

If serenity doesn't support ARM, it starts with rosetta (Intel arch) and the child subprocess that it spawns are also run under rosetta. That wouldn't pe a problem if it wouldn't be a performance drop. (Opening the browser also in intel arch took around 15s, and when we opened it with selenium in python (in Apple) it took ~4s)

If you need any other information let me know.

**Comment by wakaleo** [comment]
I don't think this is related to Serenity directly. If you are using WebDriverManager to download the drivers, then WebDriverManager takes care of what drivers are used; otherwise you configure the drivers you want in the Serenity configuration. So I don't know what you mean by "support ARM architecture for M1".

**Comment by iamlevik** [comment]
First and foremost, sorry for jumping to assumptions. In the end I've managed to solve it by using a ARM based JDK, which reduced my testing times from 38 seconds to 9.5. 

P.S. People with M1 Macbooks, make sure your JDK/JRE is ARM based. 

Thank you once again for the prompt response.
